qm.8.md

```diff
@@ -0,0 +1,121 @@
+% QM 8
+
+## NAME
+QM - Setup Containerized environment for running Functional Safety QM (Quality Management) software.
+
+## SYNOPSIS
+This package allow users to setup an environment which prevents applications
+and container tools from interfering with other all other processes on the
+system.
+
+The QM runs its own version of systemd and Podman to isolate not only the
+applications and containers launched by systemd and Podman but systemd and
+Podman themselves.
+
+Software installed into the QM environment under the /usr/lib/qm/rootfs
+directory is automatically isolated from the host. If developers need to
+further isolate their applications from other processes in the QM they should
+use container tools like Podman.
+
+## INSTALL
+
+After you have installed the software package, you need to execute the
+/usr/share/qm/setup script to setup and install the /usr/lib/qm/rootfs
+with packages to run an isolated environment. By default the script installs
+the selinux-policy-targeted, podman, systemd, hirte packages and then enables
+and runs a qm.container Podman quadlet.
+
+You can examine this quadlet with the following command:
+
+```
+systemctl status quadlet command.
+● qm.service
+     Loaded: loaded (/etc/containers/systemd/qm.container; generated)
+    Drop-In: /usr/lib/systemd/system/service.d
+             └─10-timeout-abort.conf
+     Active: active (running) since Tue 2023-04-11 15:43:45 EDT; 28min ago
+   Main PID: 993674 (conmon)
+      Tasks: 11 (limit: 76801)
+     Memory: 275.1M (swap max: 0B)
+        CPU: 4.527s
+     CGroup: /QM.slice/qm.service
+             ├─libpod-payload-00de006493bc970788d6c830beb494a58a9a2847a5eda200812d3a8b4e214814
+             │ ├─init.scope
+             │ │ └─993676 /sbin/init
+             │ └─system.slice
+             │   ├─dbus-broker.service
+             │   │ ├─993763 /usr/bin/dbus-broker-launch --scope system --audit
+             │   │ └─993764 dbus-broker --log 4 --controller 9 --machine-id 4ce4c21b211d41e78b7b64418c1c0cb5 -->
+             │   ├─systemd-journald.service
+             │   │ └─993718 /usr/lib/systemd/systemd-journald
+...
+```
+
+## CGROUPS QM.slice
+
+Notice that the QM environment is running systemd and other services within the
+QM.Slice. You can use this slice to modify the cgroups controls of all of the
+processes within the QM environment.
+
+## Install Additional packages in QM
+
+If you want to add other packages into the QM environment, use the `dnf` command
+on the host. For example if you want to install the dnf command you can execute:
+
+```
+# dnf install --installroot=/usr/lib/qm/rootfs dnf
+Updating Subscription Management repositories.
+Unable to read consumer identity
+
+This system is not registered with an entitlement server. You can use subscription-manager to register.
```

Comment:
Is this warning/error one a user would generally expect to run into?

Reply:
Not sure, this is caused by my system being registered to RHN.  So I guess I should just remove this.

Reply:
Removed.